### PR TITLE
Fix `store_failures` schema documentation

### PR DESCRIPTION
**Pull request by brunokim**
## What are you changing in this pull request and why?

I'm documenting how the config `tests.schema` is currently used. It does not define the schema where to store test failures, but the schema _suffix_ appended to the configured profile schema.

I discovered this via trial-and-error, there may be other edge cases I haven't considered.

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

**Comment by brunokim** [comment]
Hey, I see there's a conflict here, do I need to take some action?

File: website/docs/reference/resource-configs/store_failures.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Optionally set a test to always or never store its failures in the database.
 - If specified as `true` or `false`, the
 `store_failures` config will take precedence over the presence or absence of the `--store-failures` flag.
 - If the `store_failures` config is `none` or omitted, the resource will use the value of the `--store-failures` flag.
-- When true, `store_failures` save all the record(s) that failed the test only if [limit](/reference/resource-configs/limit) is not set or if there are fewer records than the limit. `store_failures` are saved in a new table with the name of the test. By default, `store_failures` use a schema named `dbt_test__audit`, but, you can [configure](/reference/resource-configs/schema#tests) the schema to a different value. 
+- When true, `store_failures` saves all records (up to [limit](/reference/resource-configs/limit)) that failed the test. Failures are saved in a new table with the name of the test. By default, `store_failures` uses the schema `{{ profile.schema }}_dbt_test__audit`, but you can [configure](/reference/resource-configs/schema#tests) the schema suffix to a different value.
```
Comment (nataliefiann) [inline website/docs/reference/resource-configs/store_failures.md:13]: Hiya @brunokim, 

Thanks for flagging this. This looks really good. I have no changes or suggestions on my end. I'll also look at the merge conflict for you.

Kind Regards
Natalie

File: website/docs/reference/resource-configs/schema.md [hunk 2]
```diff
@@ -118,14 +118,16 @@ seeds:
...
+The resulting schema is `{{ profile.schema }}_{{ tests.schema }}`, with a default suffix of `dbt_test__audit`.
+To use the same profile schema, set `+schema: null`.
 
 <File name='dbt_project.yml'>
 
 ```yml
 tests:
   +store_failures: true
-  +schema: the_island_of_misfit_tests
+  +schema: _sad_test_failures  # Will write tables to my_database.my_schema__sad_test_failures
```
Comment (nataliefiann) [inline website/docs/reference/resource-configs/schema.md:161]: Hiya @brunokim 

This is really good and great use of the example!

Kind Regards
Natalie

File: website/docs/reference/resource-configs/store_failures.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/reference/resource-configs/store_failures.md:13]: hey @brunokim and @nataliefiann , fixed up the merge conflict!

**Review by matthewshaver — APPROVED** [review]
